rmgpy/statmech/conformer.pyx

```diff
@@ -384,7 +384,8 @@ cdef class Conformer(RMGObject):
         elif pivots[0] in top1 and pivots[1] in top1:
             raise ValueError('Both pivot atoms included in top; you must specify only one pivot atom that belongs'
                              ' with the specified top.')
-
+        elif 0 in top1:
```

Comment:
a very minor comment: we don't need `elif` if the above condition raises an error
a more important comment: maybe repeat the check below for `top2` as well which could have the `0` index instead of `top1`

Reply:
elif just keeps it in one if block, which I think makes it a little bit easier to read (all the error checks separated in one block), although it makes it a bit less obvious that the checks are distinct.  

You can't run the check for top2 because top2 is inferred from top1. Essentially we only know if the input was 0 indexed if we are given a value we know we shouldn't see.